src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -402,7 +402,11 @@ const char* CryptoNative_GetOpenSslCipherSuiteName(SSL* ssl, int32_t cipherSuite
 
 int32_t CryptoNative_Tls13Supported()
 {
+#ifdef HAVE_OPENSSL_SET_CIPHERSUITES
```

Comment:
This seems perhaps trickily wrong.

If the native shim is built against OpenSSL 1.1.0 or 1.0.2 in portable mode it will see it doesn't have this function, then drop to return false; even if it was portable-bound to 1.1.1 where it should have returned true.

That is, of course, predicated on me remembering that you didn't force-set HAVE_OPENSSL_SET_CIPHERSUITES when building in portable mode.

Reply:
Oh, and if it's FALLBACK_FUNCTION now it would return true on portable-bound non-1.1.1, since it'll be asking "does the local_ function exist"

Reply:
@bartonjs yes, I've removed that before I had more context on what that was for, will revert it back